data-reconciliation-app/package.json

```diff
@@ -19,15 +20,19 @@
     "protobufjs": "^6.10.1"
   },
   "devDependencies": {
+    "@jest/globals": "^29.3.1",
```

Comment:
Let's add following dependencies for better developer productivity:

```json
"@types/jasmine": "^4.3.0",
"@types/mocha": "^10.0.0",
"@types/node": "^18.11.9",
"ts-node": "^10.9.1",
```

Also, I'm not sure about all other dev dependencies 🤔 
Do we use these anywhere in our sample app?

```json
@jest/globals
@rollup/plugin-commonjs
@rollup/plugin-node-resolve
@rollup/plugin-typescript
@types/jsrsasign
@types/lodash-es
del-cli
http-server
rollup
```

Reply:
Yeah it is used for build the app bundle that will be deployed to CCF